src/olympia/users/models.py

```diff
@@ -498,10 +497,6 @@ def ban_and_disable_related_content_bulk(cls, users, move_files=False):
         addons_sole = Addon.unfiltered.filter(id__in=addon_ids - addon_joint_ids)
         # set the status to disabled - using the manager update() method
         addons_sole.update(status=amo.STATUS_DISABLED)
-        # collect Files that need to be disabled now the addons are disabled
-        File.objects.filter(version__addon__in=addons_sole).update(
-            status=amo.STATUS_DISABLED
-        )
```

Comment:
This was obsolete - they get disabled anyway

Reply:
Wasn't it done here to force doing it by bulk? Doing it here in a single query is a lot more efficient if the user has a lot of add-ons.

Reply:
I can't remember why I did it now 🙃 but sounds plausible.

Reply:
https://github.com/mozilla/addons-server/pull/21408/commits/86b2560befafcb336a2d410758a7898fd67c0ef6